larsim/IonizationScintillation/ISCalcCorrelated.cxx

```diff
@@ -65,6 +66,9 @@ namespace larg4 {
     double EFieldStep = EFieldAtStep(detProp.Efield(), edep);
     double recomb = 0.;
 
+    //calculate recombination survival fraction value inside, otherwise zero
+
+    if(EFieldStep > 0) {
```

Comment:
Pure style comment but I'm guessing the code enclosed in this if statement should be indented?